modules/core/src/main/java/org/apache/ignite/internal/managers/communication/GridIoManager.java

```diff
@@ -235,7 +235,7 @@
  * @see IgniteMessaging
  * @see TransmissionHandler
  */
-public class GridIoManager extends GridManagerAdapter<CommunicationSpi<Serializable>> {
+public class GridIoManager extends GridManagerAdapter<CommunicationSpi<Message>> {
```

Comment:
GridIoManager is a wrapper over interface CommunicationSpi<T>. We can't use Message here, if another CommunicationSpi is in use.